xml/System.Numerics/Complex.xml

```diff
@@ -1832,6 +1832,45 @@
         <altmember cref="F:System.Numerics.Complex.Zero" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Addition">
+      <AssemblyInfo>
```

Comment:
You forgot to remove AssemblyInfo from MemberGroup.
```suggestion
```

Reply:
Why? Looking at the other usage examples of MemberGroup, they specify an assembly info.

Reply:
Well, they are harmless if you want to keep them in but they does not have any purpose within the `MemberGroup`.
See https://github.com/dotnet/dotnet-api-docs/pull/2663#discussion_r305171570; we already had this discussion.

Reply:
Ha, I started the discussion, that's funny. I didn't read the rest of the comments, which is why I didn't know the answer. Thanks for linking it.